### DISCUSSION TITLE
When do variables get updated?

### DISCUSSION THREAD

**Discussion by nixin72** [elkowar/eww#136] [Q&A] [answered]
So, of course variables will get updated at every interval that you set them to update. But sometimes those variables don't need to have the most up to date value. For example, if you have a status bar that uses eww, then it makes sense that you'd want those values updated as often as you deem reasonable. But that's not always going to be the case, like if you've got a notification manager or a system monitor or something. In that case, you'd probably only want the variables to be updated when that window is open. 

As is stands, if no window is using a variable, do they continue to get updated in the background?

**Comment by elkowar** [comment, marked as answer]
no, updates only happen while at least one window that uses them is opened.
